packages/rainbowkit/src/wallets/walletConnectors/index.ts

```diff
@@ -19,6 +20,7 @@ export {
   ledgerWallet,
   metaMaskWallet,
   omniWallet,
+  portalWallet,
```

Comment:
Please use name `ripioWallet` or `ripioPortalWallet`. Ripio namespace is preferred if this connector will support the mobile wallet as well